app/client/src/pages/Review/Review.tsx

```diff
@@ -59,19 +58,19 @@ export default function Review() {
     }
 
     if (well.regionKey == null) {
```

Comment:
the order of this isn't quite correct, it should always send you to the furthest point down the data input process that it can